pow/service.go

```diff
@@ -161,8 +161,7 @@ func (pow *Service) AssignCoinbaseTxRewards(block *types.Block, totalReward comm
 		rewardDposArbiter := common.Fixed64(math.Ceil(float64(totalReward) * 0.35))
 		rewardMergeMiner := common.Fixed64(totalReward) - rewardCyberRepublic - rewardDposArbiter
 
-		if rewards := pow.arbiters.GetArbitersRoundReward();
-			rewards != nil && len(rewards) > 0 {
+		if rewards := pow.arbiters.GetArbitersRoundReward(); rewards != nil && len(rewards) > 0 {
```

Comment:
Only `len(rewards) > 0` is ok, because `len(nil) == 0`.

Reply:
Thanks, learned a lot.